src/features/chat/handlers.ts

```
@@ -170,182 +290,296 @@ export const processAIResponse = async (
+          const delimiterIndex =
+            processableTextForSpeech.indexOf(CODE_DELIMITER)
+          if (delimiterIndex !== -1) {
+            const beforeCode = processableTextForSpeech.substring(
+              0,
+              delimiterIndex
+            )
+            const afterDelimiterRaw = processableTextForSpeech.substring(
+              delimiterIndex + CODE_DELIMITER.length
+            )
+
+            //
```

Biome (1.9.4)

[error] Unnecessary label.

Safe fix: Remove the unnecessary label.
You can achieve the same result without the label.


(lint/complexity/noUselessLabel)